File: src/backend/oracle-data-interface/src/main/java/ca/bc/gov/trafficcourtsonline/oracledatainterface/model/Dispute.java
package ca.bc.gov.trafficcourtsonline.oracledatainterface.model;

import java.sql.Date;
import java.util.List;

import javax.persistence.CascadeType;
import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.Id;
import javax.persistence.OneToMany;
import javax.persistence.Table;

import com.fasterxml.jackson.annotation.JsonManagedReference;

import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

//mark class as an Entity   
@Entity  
//defining class name as Table name  
@Table
@Getter @Setter @NoArgsConstructor
public class Dispute {
	@Id
	@GeneratedValue
	@Column
	private int id;
	@Column
	private String ticketNumber;
	@Column
	private String courtLocation;
	@Column
	private Date violationDate;
	@Column
	private String disputantSurname;
	@Column
	private String givenNames;
	@Column
	private String streetAddress;
	@Column
	private String province;
	@Column
	private String postalCode;
	@Column
	private String homePhone;
	@Column
	private String driversLicense;
	@Column
	private String driversLicenseProvince;
	@Column
	private String workPhone;
	@Column
	private Date dateOfBirth;
	@Column
	private String enforcementOrganization;
	@Column
	private Date serviceDate;
	@OneToMany(mappedBy = "dispute", cascade = CascadeType.ALL)
	@JsonManagedReference("dispute_ticket_counts")
	private List<TicketCount> ticketCounts;
    @Column
    private boolean lawyerRepresentation;
    @Column
    private String interpreterLanguage;
    @Column
    private boolean witnessIntent;
}

File: src/backend/TrafficCourts/Workflow.Service/Models/Dispute.cs

using System.Text;
using System.Text.Json.Serialization;
using TrafficCourts.Common.Utils;

namespace TrafficCourts.Workflow.Service.Models
{
    public class Dispute
    {
        public string TicketNumber { get; set; } = null!;
        public string CourtLocation { get; set; } = null!;
        public DateTime ViolationDate { get; set; }
        public string DisputantSurname { get; set; } = null!;
        public string GivenNames { get; set; } = null!;
        public string StreetAddress { get; set; } = null!;
        public string Province { get; set; } = null!;
        public string PostalCode { get; set; } = null!;
        public string HomePhone { get; set; } = null!;
        public string DriversLicence { get; set; } = null!;
        public string DriversLicenceProvince { get; set; } = null!;
        public string WorkPhone { get; set; } = null!;
        public DateTime DateOfBirth { get; set; }
        public string EnforcementOrganization { get; set; } = null!;
        public DateTime ServiceDate { get; set; }
        public List<TicketCount> TicketCounts { get; set; } = null!;
        public bool LawyerRepresentation { get; set; }
        public string InterpreterLanguage { get; set; } = null!;
        public bool WitnessIntent { get; set; }
        public string? OcrViolationTicket { get; set; }

        public override string ToString()
        {
            return GetType().GetProperties()
                .Select(info => (info.Name, Value: info.GetValue(this, null) ?? "(null)"))
                .Aggregate(
                    new StringBuilder(),
                    (sb, pair) => sb.AppendLine($"{pair.Name}: {pair.Value}"),
                    sb => sb.ToString());
        }
    }
}
